seatunnel-engine/seatunnel-engine-server/src/test/java/org/apache/seatunnel/engine/server/dag/TaskTest.java

```diff
@@ -170,6 +170,36 @@ public void testLogicalToPhysical() throws MalformedURLException {
                 physicalPlan.getPipelineList().get(0).getCoordinatorVertexList().size(), 1);
         Assertions.assertEquals(
                 physicalPlan.getPipelineList().get(0).getPhysicalVertexList().size(), 2);
+        Assertions.assertEquals(
+                physicalPlan
+                        .getPipelineList()
+                        .get(0)
+                        .getPhysicalVertexList()
+                        .get(0)
+                        .getTaskGroupImmutableInformation()
+                        .getTasksData()
+                        .size(),
+                2);
+        Assertions.assertEquals(
+                physicalPlan
+                        .getPipelineList()
+                        .get(0)
+                        .getPhysicalVertexList()
+                        .get(0)
+                        .getTaskGroupImmutableInformation()
+                        .getJars()
+                        .get(0),
+                Sets.newHashSet(new URL("file:///fake.jar")));
+        Assertions.assertEquals(
+                physicalPlan
+                        .getPipelineList()
+                        .get(0)
+                        .getPhysicalVertexList()
+                        .get(0)
+                        .getTaskGroupImmutableInformation()
+                        .getJars()
+                        .get(1),
+                Sets.newHashSet(new URL("file:///console.jar")));
```

Comment:
How to test the classloader of three groups of jar path combinations

Reply:
covered by https://github.com/apache/seatunnel/blob/dev/seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/classloader/ClassLoaderITBase.java#L69